platform/src/main/java/org/stellar/anchor/platform/event/MskEventPublisher.java

```diff
@@ -0,0 +1,29 @@
+package org.stellar.anchor.platform.event;
+
+import java.util.Properties;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+import org.stellar.anchor.platform.config.MskConfig;
+import org.stellar.anchor.util.Log;
+
+public class MskEventPublisher extends KafkaEventPublisher {
+  public MskEventPublisher(MskConfig mskConfig) {
+    super(mskConfig);
+
+    Log.debugF("MskConfig: {}", mskConfig);
+    Properties props = new Properties();
+    props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, mskConfig.getBootstrapServers());
+    props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+    props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
```

Comment:
`BOOTSTRAP_SERVERS_CONFIG`, `KEY_SERIALIZER_CLASS_CONFIG` and `VALUE_SERIALIZER_CLASS_CONFIG` are already being set in the parent when you call `super(mskConfig)`, so you can remove them from here:

```suggestion
```

Reply:
The `super(mskConfig)` does not use the same `Properties props` object. So, we should not call the `super(config)` here.

I had fixed the problem and added missing configuration values.